src/helpers/converters.ts

```diff
@@ -636,39 +649,103 @@ export function requestToOracleQuery(request: Request): OracleQueryUI {
     proposalLogIndex,
     proposalExpirationTimestamp,
   } = request;
-  const valueText = getPriceRequestValueText(proposedPrice, settlementPrice);
-  const queryTextHex = ancillaryData;
-  const queryText = safeDecodeHexString(ancillaryData);
-  const { title, description, umipUrl, umipNumber, project } = getQueryMetaData(
-    identifier,
-    queryText
-  );
+
+  const result: OracleQueryUI = {
+    id,
+    chainId,
+    oracleType,
+    oracleAddress,
+    chainName: getChainName(chainId),
+    actionType: getRequestActionType(state),
+    moreInformation: [],
+    project: "UMA",
+  };
```

Comment:
refactoring the returned result so that we dont include keys to data which is undefined